### ISSUE TITLE
Homebridge connection [Help needed]

### ISSUE THREAD

**Issue by MetaImi** [postlund/pyatv#86]
Hi 

I have successfully installed the app on a pi and it works with two atv4. However I do not really like home assistant (slow, web interface), but I have a fully working homebridge setup. Did you tried to connect pyatv to homebridge?

**Comment by postlund** [comment]
No, I have not. Currently I don't use HomeKit at all, so it's not that important to me. If you want to bypass Home Assistant, you either have to write a new plugin that interfaces with pyatv or use a plugin for something else, that in turn interfaces with pyatv (like Home Assistant does). Since Homebridge is written in Javascript, it will probably be quite tricky to use this library as it's python. Calling commands from the command line is probably easiest, but will likely be slow. So using Home Assistant is still the easiest way. I cannot say that I have any issues with Home Assistant being slow on my RPI3. It will of course be faster on a mer powerful device, but it's works good for me at least.

Bottom line is: it's not possible to use pyatv with Homebridge without some sort of plugin.

**Comment by MetaImi** [comment]
Thanks for the fast answer. I was searching how to use it from command line but I was afraid of the speed also. (Will have to try it once I have some time. ). i have noticed sometimes home assistant is out of sync with my tv. (eg: tv is playing  / assistant shows pause) Is there a way to fetch the current status / metadata?

**Comment by robbiet480** [comment]
Looks like [this](https://github.com/lprhodes/homebridge-apple-tv) is a possible solution.

**Comment by postlund** [comment]
That seems like the missing puzzle piece. But unless it implements MediaRemote protocol I doubt that it will work much better than Home assistant and pyatv for ATV4 devices.

**Comment by postlund** [comment]
(based on the wording, I believe much inspiration will be taken from this project)

**Comment by MetaImi** [comment]
It is just a placeholder yet. But I think it will be promising. BTW do you have any doc about MediaRemote protocol? So how is your library working?

**Comment by postlund** [comment]
Yes, we'll have to see where it goes. But my guess is that it will implement DAAP because that's wht everybody does. It's quite well documented too, which MediaRemore is not. The only source on that protocol that I know of is https://github.com/jeanregisser/mediaremotetv-protocol. My protocol uses DAAP. Everything is documented under docs.

**Comment by MetaImi** [comment]
Thanks :) This helps a lot.

**Comment by postlund** [comment]
Sure, NP! I guess pyatv is not really in the scope of HomeKit. So I will close this issue.
